commit ea1481d08d561cdf0229e440b70b88fe336d3f89
Author: Sage Weil <sweil@redhat.com>
Date:   Tue Jun 12 11:18:44 2018 -0500

    mds: addr -> addrvec
    
    Update MDSMap to use entity_addrvec_t instead of entity_addr_t.  Adjust
    the various mon and mgr pieces that touch it accordingly.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/Mgr.cc b/src/mgr/Mgr.cc
index 6a027deedf1..26142d84bfd 100644
--- a/src/mgr/Mgr.cc
+++ b/src/mgr/Mgr.cc
@@ -579,12 +579,12 @@ void Mgr::handle_fs_map(MFSMap* m)
 	  metadata->metadata.count("addr") == 0) {
         update = true;
       } else {
-        auto metadata_addr = metadata->metadata.at("addr");
-        const auto map_addr = info.addr;
-        update = metadata_addr != stringify(map_addr);
+        auto metadata_addrs = metadata->metadata.at("addr");
+        const auto map_addrs = info.addrs;
+        update = metadata_addrs != stringify(map_addrs);
         if (update) {
-          dout(4) << "MDS[" << info.name << "] addr change " << metadata_addr
-                  << " != " << stringify(map_addr) << dendl;
+          dout(4) << "MDS[" << info.name << "] addr change " << metadata_addrs
+                  << " != " << stringify(map_addrs) << dendl;
         }
       }
     } else {
@@ -596,7 +596,7 @@ void Mgr::handle_fs_map(MFSMap* m)
 
       // Older MDS daemons don't have addr in the metadata, so
       // fake it if the returned metadata doesn't have the field.
-      c->set_default("addr", stringify(info.addr));
+      c->set_default("addr", stringify(info.addrs));
 
       std::ostringstream cmd;
       cmd << "{\"prefix\": \"mds metadata\", \"who\": \""
